src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, {
+    useState,
+    useEffect
+  } from "react";
+  
+  import "./App.css";
+  
+  function App() {
+    const emptyTodo = {
+      title: "",
+      description: ""
+    };
+  
+    const [todos, setTodos] = useState([]);
+    const [newTodo, setNewTodo] = useState(emptyTodo);
+  
+    useEffect(() => {
+      fetch("http://localhost:3000/todos")
```

Comment:
I know that as this project's purpose is study and it's scope is tiny, you probably won't need to GET your todos in any other place but just for the sake of learning good practices, let's move all network operations like this one (basically everything calling `fetch`) and let's create a module that is responsible for consuming data from your TODO API.

You could use lots of different names for this module, like: `todoAPI.js`, `todoClient.js`, `todoSDK.js` or whatever.

I for one would do it like this: `src/api/todoClient.js` or something like that.

Inside this module you'll create functions to make the network operations you want in your API. I would design this module in a way you could use it like this:

```javascript
import todoClient from 'path/to/api/todoClient'

...

useEffect(() => {
      todoClient.get().then(todos => setTodos(todos));

...
```

if you prefer call it `todoAPI` or something, suit yourself. Could do something like:

`src/api/todo.js`

and then

```javascript
import todoAPI from 'path/to/api/todo'
```

Whatever makes you happier